com.ibm.streamsx.inet/impl/java/src/com/ibm/streamsx/inet/http/HTTPPostOper.java

```diff
@@ -389,6 +422,23 @@ void sleepABit(double seconds) throws InterruptedException {
 			Thread.sleep(100);
 		}
 	}
+	
+	boolean isAttributeToPost(String attributeName) {
+		
+		if (postAttributes.isEmpty()) {
+			trace.log(TraceLevel.INFO, "postAttriburte to use: " + attributeName);
+			return true;
+		}
+		for (Iterator<String> iterator = postAttributes.iterator(); iterator.hasNext();) {
```

Comment:
As an FYI this could be written more simply as:

```
for (String postAttribute : postAttributes) {
    ....
}
```

or even in this case:
```
postAttributes.contains(attributeName)
```

Though I think creating a Set from the list of attributes is a better approach